Cargo.toml

```diff
@@ -64,6 +64,7 @@ smlang = "0.5.1"
 enum-iterator = "1.1.3"
 rand_xorshift = "0.3.0"
 rand_core = "0.6.3"
+shared-bus-rtic = "0.2.2"
```

Comment:
Check with @ryan-summers whether this is still the preferred solution.

Reply:
I can't use `shared-bus` in rtic resources since it does not impl. send/sync. I think I can also not make it a [local initialized resource](https://rtic.rs/1/book/en/by-example/resources.html#task-local-initialized-resources) since I need to do the whole DAC setup in `setup()`. 
I've also asked @ryan-summers about this.